AlphaWallet/Core/Coordinators/Alerts/AlertDataStoreType.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  AlertDataStoreType.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 17.09.2021.
+//
+
+import UIKit
+import PromiseKit
+
+enum AlertUpdates {
+    case enabled(Bool)
+    case value(value: Double, marketPrice: Double)
+}
+
+protocol AlertDataStoreType: class {
```

Comment:
BTW, do you think it helps to create a protocol when there's only 1 class implementing it? Why?

Reply:
I found it helpfull for testing, it allows us to sapp mock objects  and so on. As well as extract interface from implementation.